templates/component/name.component.js

```diff
@@ -5,6 +5,7 @@ angular.module('<%= scriptAppName %>')
     template: '<h1>Hello {{ $ctrl.message }}</h1>',
     bindings: { message: '<' },
     controller: function() {
-      this.message = 'World'
+      var vm = this;
+      vm.message = 'World';
```

Comment:
In this Post : https://johnpapa.net/angularjss-controller-as-and-the-vm-variable/, there is some explanations for var vm = this. It seems also to apply in the component case.
But I would create a named function and not just inlining the function inside the component.
What do you think?